src/utils/changelog.ts

```diff
@@ -39,9 +39,13 @@ export interface ChangesetLoc {
   padding: string;
 }
 
+function escapeMarkdownPound(text: string): string {
+  return text.replace(/#/g, '&#35;');
+}
+
 function markdownHeader(level: number, text: string): string {
   const prefix = new Array(level + 1).join('#');
-  return `${prefix} ${text}`;
+  return `${prefix} ${escapeMarkdownPound(text)}`;
```

Comment:
YOU MEAN OCTOTHORPE :trollface:

Reply:
LOL WUT?

Reply:
https://en.wikipedia.org/wiki/Number_sign 😁

<img width="958" alt="Screen Shot 2021-09-24 at 10 07 31 AM" src="https://user-images.githubusercontent.com/134455/134688227-a44f7458-2d1f-496d-a3c0-1959ca38b263.png">